core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_prowconfig.yaml

```diff
@@ -0,0 +1,154 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.10
+    - openshift-4.11
+    - openshift-4.12
+    - openshift-4.13
+    - openshift-4.14
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.10
+    - release-4.11
+    - release-4.12
+    - release-4.13
+    - release-4.14
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
+    labels:
+    - approved
+    - backport-risk-assessed
+    - cherry-pick-approved
+    - jira/valid-bug
+    - jira/valid-reference
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    repos:
+    - openshift/coredns-ocp-dnsnameresolver
+  - includedBranches:
+    - openshift-4.15
+    - release-4.15
+    labels:
+    - approved
+    - backport-risk-assessed
+    - cherry-pick-approved
+    - jira/valid-bug
+    - jira/valid-reference
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    repos:
+    - openshift/coredns-ocp-dnsnameresolver
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - docs-approved
+    - jira/valid-reference
+    - lgtm
+    - px-approved
+    - qe-approved
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/coredns-ocp-dnsnameresolver
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - jira/valid-reference
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/coredns-ocp-dnsnameresolver
+  - excludedBranches:
+    - feature-es6x
+    - feature-prom-connector
+    - main
+    - master
```

Comment:
How all these branches relate to the dnsnamesresolver plugin?

Reply:
This is a common format for prowconfig across repos belonging to openshift. Most of the repos don't have the branches. The prowconfig files are updated through automation during branch cut and release. I am guessing the this format may have something to do with the automation.